sdk/communication/azure-communication-callautomation/src/main/java/com/azure/communication/callautomation/CallConnectionAsync.java

```diff
@@ -159,8 +157,6 @@ Mono<Response<Void>> hangUpWithResponseInternal(boolean isForEveryone, Context c
 
             return (isForEveryone ? callConnectionInternal.terminateCallWithResponseAsync(
                     callConnectionId,
-                    UUID.randomUUID(),
```

Comment:
Yes, the generated method implementations take care of corresponding API call parameters and no longer need caller to pass.

Reply:
I assume they are the `Repeatability-Request-ID` and `Repeatability-First-Sent`. If such, yes, they are handled by codegen and generated in impl (instead of exposing to user or dev).